torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -3672,16 +3672,13 @@ def parse_env(var):
                 ddp_logging_data.avg_backward_compute_comm_overlap_time)
             # test larger net with mixed data types, verify multiple bucket sizes
             model = LargeNet()
-            model.float()
-            model.fc1.double()
```

Comment:
Have you tried `model.float().fc1.double()`?

Out of curiosity, will there still be the same data race issue if change `model.float()` to `model.fc2.float()`?

Reply:
tried both, Tsan still complained:(